app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -33,7 +33,7 @@ CONFIG_INTEL_ADSP_TIMER=y
 CONFIG_MM_DRV_INTEL_ADSP_TLB_REMAP_UNUSED_RAM=y
```

Comment:
@marcinszkudlinski No need to update this PR, but the git commit messages look a bit odd. The text should be line-wrapped at 75 columns (originating from Linux coding guidelines). I.e.

"L3 memory should support have support
for allocating/freeing memory
A heap support in L3 is introduced by
this commit"

-> 

""
L3 memory should support have support for allocating/freeing
memory A heap support in L3 is introduced by this commit
""